### PR TITLE
[DEVOPS-1189] io update-proposal submit: include linux by default

### PR DESCRIPTION
**Pull request by rvl**
Linux Daedalus installers will be proposed on mainnet from now on.

### CODE REVIEW

File: docs/Daedalus-Installer-Update-Procedure.md [hunk 1]
```diff
@@ -174,12 +174,13 @@ the effect of immediately updating the download links on
...
+    io -c NETWORK.yaml update-proposal submit DATE --relay-ip 1.2.3.4
+        [--without-linux] [--without-macos] [--without-windows]
 
-By default, installers will be proposed for Windows and macOS, but not
-Linux. Use the `--with-linux` flag to include these installers in the
-update proposal. While the Linux installer is in beta, it will not be
-proposed on the mainnet blockchain.
+**Updated 2018-12-17**
+By default, installers will be proposed for Linux, Windows and macOS.
+Use the `--without-OS` flag to include these installers in the
```
Comment (angerman) [inline docs/Daedalus-Installer-Update-Procedure.md]: `without` => `include`? This is meant to be `exclude` I assume?

Comment (rvl) [reply docs/Daedalus-Installer-Update-Procedure.md]: yes that's what I meant :-) Fixed, thanks.

**Comment by rvl** [comment]
We can merge this after the 2.0.0 mainnet update proposal. The docs have been updated in the meantime.

**Pull request merged**